src/map/clif.c

```diff
@@ -25526,6 +25527,71 @@ static void clif_item_reform_result(struct map_session_data *sd, int index, enum
 #endif
 }
 
+static void clif_enchantui_open(struct map_session_data *sd, int64 enchant_group)
+{
+#if PACKETVER_MAIN_NUM >= 20210203 || PACKETVER_RE_NUM >= 20211103
+	clif->open_ui_send2(sd, ZC_ENCHANT_UI, enchant_group);
```

Comment:
State is not saved?
Does it allow to talk to other NPC/use skill/move while UI is opened?